### DISCUSSION TITLE
AvatarGroup - Show +x without actually rendering X Avatars

### DISCUSSION THREAD

**Discussion by timothychoi-rg** [chakra-ui/chakra-ui#7861] [Q&A]
Is this possible? I would like to avoid having empty DOM tags.

**Comment by HaleyWardo** [comment]
Can you show an example of your implementation? If `max` is passed in, it will only render the number of max allowed. So you shouldn't have empty DOM tags. 

```
<AvatarGroup max={2}>
  <Avatar/>
  <Avatar/>
  <Avatar/>
</AvatarGroup>
```

**Reply by timothychoi-rg** [reply]
Hi @HaleyWardo - yes, I would like to avoid even putting in the third `<Avatar>` in code.

**Reply by HaleyWardo** [reply]
If thats the case, then you will probably just want to slice the items and only return the amount of Avatars you want.

```
const avatars = avatarItems.slice(0, 2);

return avatars.length ? (
  <AvatarGroup>
    {avatars.map((avatar) => (
      <Avatar key={avatar.id} {...avatar} />
    ))}
  </AvatarGroup>
) : null;
```
